reports/README.md

```diff
@@ -16,31 +16,63 @@ $ earl-report --json --format html --template template.haml -o index.html earl.j
 ```
 
 
-<h2>Instructions for submitting implementation reports</h2>
+## Instructions for submitting implementation reports
 
-<p>Reports should be submitted in Turtle format to
+Reports should be submitted in Turtle format to
   <a href="mailto:public-rch-wg@w3.org">Public RCH WG</a> or via a Pull
-  Request to the <a href="https://github.com/w3c/rdf-canon/pulls">w3c/rdf-canon</a>.</p>
+  Request to the <a href="https://github.com/w3c/rdf-canon/pulls">w3c/rdf-canon</a>.
```

Comment:
```suggestion
  Request to the [w3c/rdf-canon repository](https://github.com/w3c/rdf-canon/pulls).
```

Reply:
I'm fine with this, but since it's largely copied from the Haml (HTML) file, and HTML is compatible with Markdown, I didn't feel the need. But, when reading the file as Markdown, it is easier for a human to parse.